src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -1002,27 +1002,6 @@ private void startIce4j(
                 }
             }
 
-            // These properties are moved to ice4j. This is to make sure that we
-            // still support the old names.
-            String oldPrefix = "org.jitsi.videobridge";
-            String newPrefix = "org.ice4j.ice.harvest";
-            for (String propertyName : new String[]{
-                HarvesterConfiguration.NAT_HARVESTER_LOCAL_ADDRESS,
-                HarvesterConfiguration.NAT_HARVESTER_PUBLIC_ADDRESS,
-                HarvesterConfiguration.DISABLE_AWS_HARVESTER,
-                HarvesterConfiguration.FORCE_AWS_HARVESTER,
-                HarvesterConfiguration.STUN_MAPPING_HARVESTER_ADDRESSES})
-            {
-                String propertyValue = cfg.getString(propertyName);
-
-                if (propertyValue != null)
-                {
-                    String newPropertyName
-                        = newPrefix
-                                + propertyName.substring(oldPrefix.length());
-                    System.setProperty(newPropertyName, propertyValue);
```

Comment:
So we don't need to inject these as system properties anymore?  I thought we still did.

Reply:
The code which reads `org.ice4j` properties from our file and injects them as system properties still exists. The removed code reads `org.videobridge` properties and injects them as `org.ice4j`. The properties have been marked deprecated for a long time, and I verified we don't use them anywhere.